build.gradle

```diff
@@ -20,7 +20,7 @@ nexusStaging {
 }
 
 dependencies {
-    api 'com.synopsys.integration:issuetracker-common:0.0.1'
+    api 'com.synopsys.integration:issuetracker-common:0.0.2-SNAPSHOT'
```

Comment:
I will update this to the released version when issuetracker-common is merged and released.